src/platform/site-wide/login/components/SignInModal.jsx

```diff
@@ -44,6 +45,18 @@ class SignInModal extends React.Component {
               <h2>One site. A lifetime of benefits and services at your fingertips.</h2>
             </div>
           </div>
+          <div className="downtime-notification row">
+            <div className="columns usa-width-one-half medium-6">
+              <div className="form-warning-banner">
+                <AlertBox
+                  headline={<h3>DS Logon isn't working quite right</h3>}
+                  content={<p>If you're having trouble signing in to Vets.gov using your DS Logon ID and password, please try again later. Or, you can try signing in with your MyHealth_e_Vet ID and password or through ID.me.</p>}
```

Comment:
The `p` element I don't think should be necessary, but doesn't hurt anything either way.